h/api/test/views_test.py

```diff
@@ -258,3 +224,362 @@ def _assert_event_published(action):
         'delete': ['alice'],
     },
 }
+
+
+class TestSearch(object):
```

Comment:
We should be consistent about this. No classes in tests. I get what you're saying about grouping, but it has downsides and visual noise such as the useless `self` parameters and the pylint pragma.